test/test_priority_queue.jl

```diff
@@ -110,12 +110,14 @@ import Base.Order.Reverse
     @testset "PriorityQueueMethods" begin
         pq1 = PriorityQueue('a'=>1, 'b'=>2)
 
-        @testset "peek/get/dequeue!" begin
+        @testset "peek/get/dequeue!/get!" begin
             @test peek(pq1) == ('a'=>1)
             @test get(pq1, 'a', 0) == 1
             @test get(pq1, 'c', 0) == 0
             @test dequeue!(pq1) == 'a'
             @test dequeue!(pq1) == 'b'
+            @test get!(pq1, 'c', 0) == 0
+            @test peek(pq1) == ('c'=>0)
```

Comment:
This also needs a test that when the iterm is present, `get!` returns the correct result